pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -101,6 +101,25 @@ def test_to_datetime_format_YYYYMMDD(self, cache):
         expected = Series(["20121231", "20141231", "NaT"], dtype="M8[ns]")
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
```

Comment:
can you parameterize over [None, np.nan, pd.NaT, 'NaT'] here  in place of the None